specs/APIReview_TextureStream.md

```diff
@@ -0,0 +1,592 @@
+  /// corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2StagingTextureStream* sender,
+      [in] ICoreWebView2StagingTextureStreamWebTextureReceivedEventArgs* args);
+}
+
+/// This is the event args interface for web texture.
+[uuid(a4c2fa3a-295a-11ed-a261-0242ac120002), object, pointer_default(unique)]
+interface ICoreWebView2StagingTextureStreamWebTextureReceivedEventArgs : IUnknown {
+  // Return ICoreWebView2StagingWebTexture object.
+  // The call does not create new ICoreWebView2StagingWebTexture object, instead
+  // returns the same object.
+
+  // The shared buffer handle will be reused when ICoreWebView2StagingWebTexture
+  // object is released. So, the host should not refer handle or resource of
+  // the ICoreWebView2StagingWebTexture after its release.
+
+  HRESULT GetWebTexture([out, retval] ICoreWebView2StagingWebTexture** value);
+}
+
+/// Texture received stream buffer that the renderer writes to so that the host
+/// will read on it.
+[uuid(b94265ae-4c1e-11ed-bdc3-0242ac120002), object, pointer_default(unique)]
+interface ICoreWebView2StagingWebTexture : IUnknown {
+  /// Texture buffer handle. The handle's lifetime is owned by the
+  /// ICoreWebView2StagingTextureStream object so the host must not close it.
+  /// The same handle value will be used for same buffer so the host can use
+  /// handle value as a unique buffer key.
+  /// If the host opens its own resources by handle, then it is suggested
+  /// that the host removes those resources when the handle's texture size
+  /// is changed because the browser also removed previously allocated different
+  /// sized buffers when image size is changed.
+  [propget] HRESULT Handle([out, retval] HANDLE* handle);
+
+  /// Texture buffer resource.
+  /// The same resource value will be used for same buffer so the host can use
+  /// resource value as a unique buffer key.
+  /// ICoreWebView2StagingTextureStream object has a reference of the resource
+  /// so ICoreWebView2StagingWebTexture holds same resource object for
+  /// the same buffer.
+  [propget] HRESULT Resource([out, retval] IUnknown** resource);
+
+  /// It is timestamp of the web texture. Javascript can set this value
+  /// with any value, but it is suggested to use same value of its original
+  /// video frame that is a value of SetBuffer so that the host is able to
+  /// tell the receiving texture delta.
+  [propget] HRESULT Timestamp([out, retval] ULONGLONG* value);
+}
+
+/// This is the callback for web texture stop.
+[uuid(77eb4638-2f05-11ed-a261-0242ac120002), object, pointer_default(unique)]
+interface ICoreWebView2StagingTextureStreamWebTextureStreamStoppedEventHandler : IUnknown {
+  /// Called to provide the implementer with the event args for the
+  /// corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2StagingTextureStream* sender,
+      [in] IUnknown* args);
+}/// This is the callback for web texture.

```

Comment:
All of the interfaces after this line are duplicates of the above

Reply:
I'm sorry for confusion.